spec.bs

```diff
@@ -351,16 +359,16 @@ spec: html; urlPrefix: https://www.rfc-editor.org/rfc/
     1. For each |epoch| in |epochs|:
         1. If |epoch|'s [=epoch/top 5 topics with caller origins=] is empty (implying the topics calculation failed for that epoch), then continue.
         1. Let |topic| be null.
-        1. Let |topTopicIndexDecisionHash| be64 bit truncation of the output of the [=HMAC algorithm=], given input parameters: whichSha=SHA256, key=user agent's [=user agent/user topics state=]'s [=user topics state/hmac key=], and message_array=the concatenation of "top-topic-index-decision|", |epoch|'s [=epoch/time=], and |callerContext|'s [=topics caller context/top level context domain=]
+        1. Let |topTopicIndexDecisionHash| be 64 bit truncation of the output of the [=HMAC algorithm=], given input parameters: whichSha=SHA256, key=user agent's [=user agent/user topics state=]'s [=user topics state/hmac key=], and message_array=the concatenation of |topTopicIndexDecisionHash|, |epoch|'s [=epoch/time=], and |callerContext|'s [=topics caller context/top level context domain=]
```

Comment:
I don't understand how this variable is an input into its own calculation.

Maybe this could be split into multiple steps? E.g. concatenation, HMAC, truncation?

Reply:
You are right. The original parameter is actually right and I mistakenly changed that... I split it into multiple steps.